### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by AnuragSahu11**
Capture

### CODE REVIEW

**Review by raghav2110 — COMMENTED** [review]
The code looks good. Just a suggestion from my side you can try to maintain one unit across if possible.

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+.avatar-xs {
+  position: relative;
+  vertical-align: middle;
+  font-size: 4.4px;
+}
+.avatar-xs img {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
```
Comment (lannisterrr) [inline components/avatar.css:15]: a suggestion, you are using a border-radius property everywhere, you can also create a separate class specifically for this.

Comment (AnuragSahu11) [reply components/avatar.css:15]: Yes

File: components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  height: 120px;
+  border-radius: 50%;
+  border: solid 3px var(--primary);
+}
+
+/* avatar with badge */
+
+.avatar-badge {
+  position: absolute;
+  border: solid white;
```
Comment (bhanuprakashj) [inline components/avatar.css:80]: Here border width is not mentioned. is the color white coming from variables? if not, it is better to add from variables.
Generally border shorthand property goes like
border: border-width border-style border-color
I have seen it is written as border: border-style border-width border-color
Reference: [Shorthand border properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#border_properties)

**Pull request merged**